BenchmarkNdkSample/benchmark/build.gradle

```diff
@@ -2,22 +2,33 @@ apply plugin: 'com.android.library'
 apply plugin: 'androidx.benchmark'
 
 android {
-    compileSdkVersion 28
+    compileSdkVersion 29
 
     defaultConfig {
         minSdkVersion 14
-        targetSdkVersion 28
+        targetSdkVersion 29
         versionCode 1
         versionName "1.0"
 
-        testInstrumentationRunner "androidx.benchmark.junit4.AndroidBenchmarkRunner"
+        // Change this to enable profiling, can be one of:
+        //   * None
+        //   * Sampled
+        //   * Method
+        //   * ConnectedSampled
+        //   * ConnectedAllocation
+        // See available options and full descriptions at: d.android.com/benchmark#profiling
+        testInstrumentationRunnerArgument 'androidx.benchmark.profiling.mode', 'none'
```

Comment:
remove this argument and comment from the Ndk sample specifically - it won't be able to profile NDK code, since it can only capture java/kotlin stacks

Reply:
Done